docs/source/extension/extension_dev.rst

```diff
@@ -47,6 +47,7 @@ We provide a set of guides to get started writing extensions for JupyterLab:
 - :ref:`extension_tutorial`: A tutorial to learn how to make a simple JupyterLab extension.
 - The `JupyterLab Extension Examples Repository <https://github.com/jupyterlab/extension-examples>`_: A short tutorial series to learn how to develop extensions for JupyterLab by example.
 - :ref:`developer-extension-points`: A list of the most common JupyterLab extension points.
+- :ref:`Lab 4 / Notebook 7 compatible extensions <dual_compatible_extensions>` For developing extensions that work in both JupyterLab and Jupyter Notebook 7
```

Comment:
Should this new page be added to the sidebar, so it's easier to find?

![image](https://github.com/jupyterlab/jupyterlab/assets/591645/605d311b-35c1-4936-b39b-7e46fcdc02de)

Reply:
It should, do you know how? I've been poking around a bit but haven't found out yet...

Reply:
Okay, I got it working. For others: I just needed to add the page name (filename no ext.) to the toctree directive for the extension_dev page (near the top extension_dev.rst):

```
.. toctree::
   :maxdepth: 1

   extension_points
   ui_components
   documents
   notebook
   virtualdom
   ui_helpers
   internationalization
   identity
   extension_tutorial
   extension_dual_compatibility
   extension_migration
```